docs/plugins/plugin-events/create-entity-controller-base.md

```diff
@@ -0,0 +1,64 @@
+---
+id: CreateEntityControllerBase
+title: Create Entity Controller Base
+sidebar_label: Create Entity Controller Base
+slug: /plugins/plugin-events/CreateEntityControllerBase
+---
+
+# Create Entity Controller Base
+
+## Description
+
+Creates the base controller of an entity
+
+## Event Name:
+`CreateEntityControllerBase`
+
+## Event Params
+
+```tsx
+export interface CreateEntityControllerBaseParams extends EventParams {
+  template: namedTypes.File;
+  entity: Entity;
+  entityName: string;
+  entityType: string;
+  entityServiceModule: string;
+  templateMapping: { [key: string]: any };
+  controllerBaseId: namedTypes.Identifier;
+  serviceId: namedTypes.Identifier;
+}
+```
+
+### template
+A template file that is used to generate the service
+This is the default template that is used for this event.
+https://github.com/amplication/amplication/blob/master/packages/amplication-data-service-generator/src/server/resource/service/service.template.ts
```

Comment:
@yuval-hazaz do we need to provide here an example from the controller instead?

Reply:
resolve